packages/sync/src/y-webrtc/y-webrtc.js

```diff
@@ -2,450 +2,570 @@
 // added to the following vars/functions: signalingConns,rooms, publishSignalingMessage, log.
 /* eslint-disable eslint-comments/disable-enable-pair */
 /* eslint-disable eslint-comments/no-unlimited-disable */
-/* eslint-disable */ 
+/* eslint-disable */
```

Comment:
Same as this file. If there are no functional changes, I think we should do a `--no-verify` or get a different PR in quickly that contains only the linting changes.